e2e-tests/websocketTests.js

```diff
@@ -0,0 +1,28 @@
+const { expect } = require('chai');
+const WebSocket = require('ws');
+
+describe('Make websocket request', () => {
+
+    it('should output success exit code', async () => {
+        const options = {
+            rejectUnauthorized: false
+        };
+
+        const serverUrl = 'wss://localhost:443/hello';
+
+        const socket = new WebSocket(serverUrl, options);
+
+        socket.on('open', () => {
+            console.log('Connected to WebSocket server');
+            socket.on('message', (data) => {
+                console.log('Received message from server:', data.toString('utf8'));
+                socket.send('test connection 1');
+                expect(data.toString('utf8')).to.equal("SERVER RECIEVED MESSAGE: test connection 1");
+                socket.send('test connection 2');
+                expect(data.toString('utf8')).to.equal("SERVER RECIEVED MESSAGE: test connection 2");
+            });
```

Comment:
Does this work? Everytime we send a message, the server will reply and this callback will run again no?

Reply:
Yeah the e2es are running locally. There's an env var bug thats breaking them in CI

Reply:
Ah sorry yeah the expects weren't running so it looked like it was passing